### PR TITLE
Fix bug where Sweden::Validator.valid_serial_number_length? would raise

### PR DESCRIPTION
**Pull request by isaacseymour**
@greysteil could you review? Also fixes a warning about a duplicate key when running on Ruby 2.2.

### CODE REVIEW

**Comment by isaacseymour** [comment]
Or actually, @matt-thomson ?

File: lib/ibandit/german_details_converter.rb [hunk 1]
```diff
@@ -657,138 +657,138 @@ class Rule003101 < BaseRule
...
-        '683' => '71020072', '684' => '70020270', '685' => '70020270',
-        '686' => '70020270', '687' => '70020270', '688' => '70020270',
-        '689' => '70020270', '690' => '76520071', '692' => '70020270',
-        '693' => '73420071', '694' => '70021180', '695' => '70320090',
-        '696' => '74320073', '697' => '54020090', '698' => '73320073',
-        '710' => '30220190', '711' => '70020270', '712' => '10020890',
-        '714' => '76020070', '715' => '75020073', '717' => '74320073',
-        '718' => '87020086', '719' => '37020090', '720' => '30220190',
-        '723' => '77320072', '733' => '83020087', '798' => '70020270'
+        '168' => '70320090', '169' => '79020076', '170' => '70020270',
```
Comment (greysteil) [inline lib/ibandit/german_details_converter.rb]: What's the deal with this chagne?

Comment (isaacseymour) [inline lib/ibandit/german_details_converter.rb]: Fixes a warning about a duplicate key when running on Ruby 2.2. (key `'167'` was in here twice before)

Comment (greysteil) [inline lib/ibandit/german_details_converter.rb]: Ah, :+1:

**Comment by greysteil** [comment]
:+1:

**Pull request merged**